### PR TITLE
feat: warn when not ingesting recordings

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

If you set an authorized URL for recordings, and don't realise the impact, or don't realise it doesn't match all of your traffic. Then we do not record any non-matching domains. This could be confusing.

## Changes

* converts the authorized URL listing to HogQL
* Shows a warning on the replay page if appropriate

## How did you test this code?

👀 locally

### CODE REVIEW

**Review by daibhin — APPROVED** [review]

**Comment by pauldambra** [comment]
@joethreepwood @Lior539 would be interested in your feedback on the words

> You have authorized domains configured for session recordings. Only recordings from these domains will be allowed. But you have 2 domains that are not authorized which are trying to send recordings. You should review your configuration.

We had a support ticket where someone had set `https://domain.com` as the authorized domain but their site was actually `https://www.domain.com` so we silently don't record anything from their site. This is intended to prompt people in that situation to check and fix their config

**Comment by joethreepwood** [comment]
Not a blocker, but we could probably just shorten this to: 

`You have unauthorized domains trying to send recordings. To accept recordings from these domains, please check your config.`
